Sources/Widgets/Representations/ArrowHandleRepresentation/index.js

```diff
@@ -74,24 +108,33 @@ function vtkArrowHandleRepresentation(publicAPI, model) {
   model.displayMapper.setInputConnection(publicAPI.getOutputPort());
   publicAPI.addActor(model.displayActor);
   model.alwaysVisibleActors = [model.displayActor];
+  model.pipelines = {
+    arrow: {
+      source: publicAPI,
+      glyph: vtkSphereSource.newInstance(),
```

Comment:
if you have a valid glyph, then must update `model.currentShape` accordingly

Reply:
here i dont have access to the state of my representation yet. The trick i used was to set a default shape that would be overriden by the 1st call of requestData. The solution i originally came up with was to include model shape in initialValues of my representation in the lineWidget/index.js level. Should i use this solution?